<?xml version="1.0" encoding="UTF-8"?><testrun name="UserServiceTest.testGet (1)" project="web-advancedQuery04" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testcase name="testGet" classname="cn.wolfcode.shop.test.UserServiceTest" time="0.667">
    <error>org.apache.ibatis.exceptions.PersistenceException: &#13;
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '= 13' at line 1&#13;
### The error may exist in cn/wolfcode/shop/mapper/UserMapper.xml&#13;
### The error may involve defaultParameterMap&#13;
### The error occurred while setting parameters&#13;
### SQL: select * from t_user id = ?&#13;
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '= 13' at line 1&#13;
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:23)&#13;
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:104)&#13;
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:95)&#13;
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectOne(DefaultSqlSession.java:59)&#13;
	at cn.wolfcode.shop.dao.impl.UserDAOImpl.get(UserDAOImpl.java:38)&#13;
	at cn.wolfcode.shop.service.impl.UserServiceImpl.get(UserServiceImpl.java:24)&#13;
	at cn.wolfcode.shop.test.UserServiceTest.testGet(UserServiceTest.java:37)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:498)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '= 13' at line 1&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)&#13;
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#13;
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)&#13;
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)&#13;
	at com.mysql.jdbc.Util.getInstance(Util.java:387)&#13;
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)&#13;
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)&#13;
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)&#13;
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2478)&#13;
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2625)&#13;
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2551)&#13;
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1861)&#13;
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1192)&#13;
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.query(PreparedStatementHandler.java:56)&#13;
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.query(RoutingStatementHandler.java:70)&#13;
	at org.apache.ibatis.executor.SimpleExecutor.doQuery(SimpleExecutor.java:57)&#13;
	at org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:267)&#13;
	at org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:141)&#13;
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:105)&#13;
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:81)&#13;
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:101)&#13;
	... 28 more&#13;
&#13;
</error>
  </testcase>
</testrun>
